src/snowflake/snowpark/_internal/open_telemetry_prototype.py

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+#
+
+import inspect
+from typing import Dict, List, Union, Any, Tuple
+
+# this parameter make sure no error when open telemetry is not installed
+open_telemetry_found = True
+try:
+    from opentelemetry import trace
+    from opentelemetry.sdk.resources import SERVICE_NAME, Resource
+
+except ImportError:
+    open_telemetry_found = False
+
+if open_telemetry_found:
+    tracer = trace.get_tracer("snowflake.snowpark.dataframe")
```

Comment:
Let's replace `snowflake` to `snow`? @sfc-gh-enie what do you think?

Reply:
+1, based on existing naming convention `snow` is the reserved namespace for snowflake attributes,  https://docs.snowflake.com/en/developer-guide/logging-tracing/event-table-columns#resource-attributes-column
we should keep it consistent across server and client.

Reply:
got it